torch/csrc/jit/passes/normalize_ops.cpp

```diff
@@ -15,7 +15,10 @@ static const std::unordered_map<Symbol, Symbol> alias_map = {
     {aten::linalg_det, aten::det},
     {aten::outer, aten::ger},
     {aten::arccosh, aten::acosh},
-    {aten::arccosh_, aten::acosh_}};
+    {aten::arccosh_, aten::acosh_},
+    {aten::trunc, aten::fix},
+    {aten::trunc_, aten::fix_},
```

Comment:
I found that if we always add commas and have the trailing `}` on the next line, that subsequent diffs get smaller.

Reply:
Sounds good if the linter doesn't complain, but the map is alias->original, so you'll need to flip these entries:

```
{aten::fix, aten::trunc}
```